nvm.sh

```diff
@@ -1235,6 +1235,8 @@ nvm_get_arch() {
     EXIT_CODE=$?
     if [ $EXIT_CODE -ne 0 ]; then
       HOST_ARCH=$(isainfo -n)
+    else
+      HOST_ARCH=$(echo "$HOST_ARCH" | tail -1)
```

Comment:
thanks! do you think it's better to do the `tail -1` on line 1234, when `pkg_info` is first called?